packages/react-scripts/scripts/init.js

```diff
@@ -118,6 +118,10 @@ module.exports = function(appPath, appName, verbose, originalDirectory, template
     cdpath = appPath;
   }
 
+  // Change command to yarn instead of yarnpkg
+  if (useYarn)
+    command = 'yarn';
```

Comment:
I'd prefer we create a separate variable, e.g. `displayedCommand`, and set it conditionally.